Main.java

```diff
@@ -1,15 +1,105 @@
+import java.util.Scanner;
+
 /**
- *
- * @author 
+ *this program tells users their total for ordering a pizza
+ * @authorTolga Nurtekin
  */
 public class Main {
 
   /**
    * The method that is executed when you hit the run button.
+   * 
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // create scanner for user input
+    Scanner input = new Scanner(System.in);
+
+    // ask user for pizza size
+    System.out.println("What size would you like you pizza in cm?");
+    // get integer from user
+    double size = input.nextDouble();
+
+    // calculate the order
+    if (size < 20) {
+
+      System.out.println("We are going to make you a cute little pizza");
+
+      double labour = size * 0.75;
+
+      double oven = size * 0.99;
+
+      double ingredients = size * 0.50;
+
```

Comment:
The only thing size needed to be multiplied by was the 0.5 per topping. Everything else should have just been added to the subtotal.

subtotal = size *0.5 + powerCost + labourCost. Also, the numbers should really be constants instead of "magic numbers"